cmd/root.go

```diff
@@ -79,6 +79,7 @@ func init() {
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "verbose output")
 	RootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "debug mode")
 	RootCmd.PersistentFlags().IntVar(&threads, "threads", 4, "number of threads to execute with")
+	viper.BindPFlag("threads", RootCmd.PersistentFlags().Lookup("threads"))                                               //Update to make sure viper binds to the flag
```

Comment:
This was specifically because I realized that thread settings set from the command line were not overriding data in the config file from `bbi init`. Probably need to actually go through and make sure verbose, debug etc are also bound to viper